tests_thien/pages/dashboard/main_page.py

```diff
@@ -0,0 +1,51 @@
+from selenpy.element.base_element import BaseElement
+from selenpy.support import browser
+from selenpy.support.conditions import have
+from selenpy.element.text_box import TextBox
+from selenpy.element.combo_box import ComboBox
+from tests_thien.utilities import constant
+
+class DashBoardMainPage(BaseElement):
+    
+    _btn_global_setting = BaseElement("class=mn-setting")
+    _btn_global_setting_item = BaseElement("xpath=//li[@class='mn-setting']//a[text()='%s']")
+    _txt_page_name = TextBox("id=name")
+    _drp_parent_name = ComboBox("id=parent")
+    _btn_ok = BaseElement("xpath=//input[@id='OK']")
+    _link_page_item= BaseElement("xpath=//div[@id='main-menu']//a[text()='%s']")
+    _items_drp = BaseElement("xpath=//option[normalize-space()='%s']")
+    items = []
+    
+    def __init__(self):
+        pass
+    
+    def check_main_page_appears(self):
+        browser.wait_until(have.title(constant.DA_TITLE+" - Execution Dashboard"))
+        return self._driver.title == constant.DA_TITLE+" - Execution Dashboard"
+    
+    def select_global_setting_btn(self,btn_name):
+        self._btn_global_setting.wait_for_visible(10)
```

Comment:
don't hard 10 as timeout. define the variable for this

Reply:
updated WAIT constant variable in constant module